modules/SofaConstraint/LCPConstraintSolver.h

```diff
@@ -168,6 +168,8 @@ class SOFA_CONSTRAINT_API LCPConstraintSolver : public ConstraintSolverImpl
     Data<double> showCellWidth;
     Data<defaulttype::Vector3> showTranslation;
     Data<defaulttype::Vector3> showLevelTranslation;
+    Data< sofa::helper::vector<double> > delta;
```

Comment:
Thank you for your comment... 
My bad, again something that should not be included in the Pull Request